Modules/BarcodeScanner/BarcodeScannerView.swift

```diff
@@ -67,7 +75,7 @@ public final class BarcodeScannerView: StatefulView<BarcodeScannerViewModel> {
 
         captureSessionQueue.suspend()
         captureSessionQueue.async { [weak self] in
-            guard let self = self else { return }
+            guard let self = self, !self.isDeinited else { return }
```

Comment:
Is this the only case where a guard is necessary?

Reply:
I thought so at first, but now I found another case where that would be needed.

To make things explicit, I now added the `isDeinited` check everywhere where a block is added to the `captureSessionQueue`.